verilog/tools/kythe/kythe_facts_extractor.cc

```diff
@@ -697,29 +692,17 @@ void KytheFactsExtractor::ExtractFunctionOrTaskCall(
     const IndexingFactNode& function_call_fact_node) {
   const auto& anchors = function_call_fact_node.Value().Anchors();
 
-  // In case function_name();
-  if (anchors.size() == 1) {
-    const auto& function_name = anchors[0];
-
-    const std::vector<const VName*> function_vnames =
-        scope_resolver_->SearchForDefinitions({function_name.Value()});
-
-    if (function_vnames.empty()) {
-      return;
-    }
+  // Search for member hierarchy in the scopes.
+  const std::vector<const VName*> definitions =
```

Comment:
variable name: definitions? or references?

Reply:
definitions because we look for the definitions not the references.